ui/helpers/constants.js

```diff
@@ -339,3 +339,5 @@ export const genericErrorMessage = 'Something went wrong';
 // Datetime format used for pushes listed in https://hg.mozilla.org/
 // e.g. Sun Feb 14 22:40:03 2021 +0000
 export const mercurialDatetimeFormat = 'ddd MMM DD HH:mm:ss YYYY ZZ';
+
+export const datetimeFormat = 'ddd MMM DD HH:mm YYYY';
```

Comment:
```suggestion
export const alertsViewDatetimeFormat = 'ddd MMM DD HH:mm YYYY';
```
Just in case we'll have more datetime formats in the future, perhaps it will be best to not be so general with the name?